flink-runtime/src/main/java/org/apache/flink/runtime/scheduler/adaptive/AdaptiveScheduler.java

```diff
@@ -279,14 +279,13 @@ public AdaptiveScheduler(
         final MetricOptions.JobStatusMetricsSettings jobStatusMetricsSettings =
                 MetricOptions.JobStatusMetricsSettings.fromConfiguration(configuration);
 
-        deploymentTimeMetrics =
-                new DeploymentStateTimeMetrics(jobGraph.getJobType(), jobStatusMetricsSettings);
+        metricsWrapper = new MetricsWrapper(jobGraph.getJobType(), jobStatusMetricsSettings);
```

Comment:
The variable needs a better name, similar to the class.

Reply:
Thanks for fixing it!